src/components/Card/IndicatorCard/IndicatorCard.js

```diff
@@ -12,23 +13,16 @@ import stripPrefixAndSuffix from '../../../helpers/stripPrefixAndSuffix';
 import getColorVariant from '../../../helpers/getColorVariant';
 import style from './IndicatorCard.scss';
 
-const classnames = require('classnames/bind').bind(style);
+// we don't create a new component here because the number styles are
+// tightly bound to the styles of the card they're in
```

Comment:
Fair enough that a new component isn't required, but is there a way to enforce or at least document the object passed to the function? I see JS doesn't have named args, but could you use parameter destructuring...?

Reply:
Yeah destructuring is our named args. Maybe we add documentation to this, but as is it's like all other functions that aren't documented. So maybe we do a JSDoc task at some point.